corehq/apps/registration/static/registration/js/new_user.ko.js

```diff
@@ -47,6 +48,8 @@ var formViewModel = function (defaults, containerSelector, steps) {
     // ---------------------------------------------------------------------
     // Step 1 Fields
     // ---------------------------------------------------------------------
+    const serverLocationModel = serverLocationSelect({});
+    self.serverLocation = serverLocationModel.serverLocation;
```

Comment:
`self.serverLocation` is what allows the data-bind from radio checkbox in the first commit to work, right? Prior to this commit, I'd imagine that commit would throw a knockout error because it couldn't find serverLocation?